charts/openyurt/templates/yurthub-staticpod.yaml

```diff
@@ -0,0 +1,104 @@
+apiVersion: apps.openyurt.io/v1alpha1
+kind: StaticPod
+metadata:
+  name: yurt-hub
+  namespace: kube-system
```

Comment:
kube-system --> {{ include "openyurt.namespace" . }}

Reply:
The name and namespace of StaticPod will affect the name of the generated comfigmap. If the user is allowed to specify it, it may be necessary to add parameters to the join command to enable [this part](https://github.com/y-ykcir/openyurt/blob/d570989c634c647904dee074af4f031781a80c08/pkg/yurtadm/util/kubernetes/kubernetes.go#L498) to find the corresponding configmap

Reply:
@y-ykcir i think maybe we need to add `--namespace` parameter for `yurtadm join` to specify namespace.